README.md

```diff
@@ -125,20 +107,49 @@ gh.serialize(format='ttl')
 
 ## Outputs
 
-The default output is JSON-ld, a JSON serialization of the [RDF](https://en.wikipedia.org/wiki/Resource_Description_Framework) data model. We follow the schema recommended by [codemeta](https://codemeta.github.io/).
-Supported formats are json-ld, turtle and n-triples.
+The default output is [Turtle](https://www.w3.org/TR/turtle/), a textual syntax for [RDF](https://en.wikipedia.org/wiki/Resource_Description_Framework) data model. We follow the schema recommended by [codemeta](https://codemeta.github.io/).
+Supported formats are turtle, json-ld and n-triples (by specifying the `--format` argument in your call i.e. `gimie data https://github.com/numpy/numpy --format 'ttl'`).
+
+With no specifications, Gimie will print results in the terminal. Want to save Gimie output to a file? Add your file path to the end : `gimie data https://github.com/numpy/numpy > path_to_output/gimie_output.ttl`
 
 ### Limitations
 
-* Currently, gimie will only the first 100 contributors of a repository (in arbitrary order), and for each users, at most 100 affiliations.
-* If a Github repository is owned by an organization, all "mentionable users" are reported as contributors. This will include all members of the organization in addition to contributors.
+* For a Github repository extraction: if the GitHub repository is owned by an organization, all "mentionable users" are reported as contributors. This will include all members of the organization in addition to contributors.
```

Comment:
This limitation has fortunately been addressed by https://github.com/SDSC-ORD/gimie/pull/33#issuecomment-1476652928. Should we then drop the `Limitations` section?

Reply:
yep sounds good